.ansible-lint

```diff
@@ -0,0 +1,14 @@
+---
+
+profile: production
+
+use_default_rules: true
+
+offline: true
+
+exclude_paths:
+  - terraform/
+  - docs/
+  - kubernetes/
+  - .yamllint.yml
+  - .apply-terraform.yml
```

Comment:
since ansible related files are in ansible folder, can we specify to the workflow that we want to run the ansible folder only ? It would maybe be better since we might not always reminder to add files and folders to this file when adding them to root directory.